styleguide/derek/templates/landing-page/landingpage-tradeshow-event.ejs

```diff
@@ -0,0 +1,104 @@
+          to drive business results through continuous IT modernization
+          over dinner.
+        </h3>
+      </div>
+      <div class="rsLpEvent-announcementRiverDetails">
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary">
+            Matsuhisa<br/>
+            98 Steele St<br/>
+            Denver, CO<br/>
+            80206<br/>
+          </span>
+        </div>
+        <div class="rsLpEvent-announcement-col">
+          <span class="rsLpEvent-announcementRiver-secondary rsLpEvent-announcementRiver-date">
+            Thursday, April 11<br/>
+            6 - 8:30 PM
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div>
+  <div class="event">
+    <div class="event-row">
+      <div class="event-details">
+        <div class="event-detailsTitle">Details</div>
+        <p>
+          Join us at Rackspace::Solve San Francisco, a free conference where you’ll learn how the latest cloud
+          solutions can help you scale your business, get your products to market quickly, and reduce the cost
+          of innovation.
+        </p>
+        <p>
+          Blah blah blahddy blah
+          <ul>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+            <li>I'm a list item</li>
+          </ul>
+        </p>
+        <p>
+          Beyond empowering companies to tap into the power of the cloud, Rackspace also strives for social
+          empowerment. In that spirit, we’ll make a monetary donation of $15 per attendee to Girls Who Code –
+          up to $5,000. In addition to a great day of learning and networking, your attendance at Rackspace::Solve
+          will help give high school girls the skills and resources they need to pursue opportunities in computing
+          fields.
+        </p>
+      </div>
+
+      <div class="event-img">
+        <div class="lp-event-img" style="background-image:url('../imgs/derek/cx-teamwork.jpg');"></div>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="event-moderators-section">
+    <div class="event-moderators-container">
+      <h3 class="event-moderators-header">Moderators</h3>
+      <% for(var i = 0; i < 2; i++) { %>
+        <div class="event-moderators-item">
+          <div class="event-moderators-picContainer">
+            <img class="event-moderators-pic" src="../imgs/derek/speaker-thumb.jpg" alt="">
+          </div>
+          <div class="event-moderators-info">
+            <h3 class="event-moderators-name">Derek Zoolander</h3>
+            <p>Supermodel, Los Angeles</p>
+            <p>Mugatu Inc.</p>
+            <p>
+              Street art dreamcatcher cronut typewriter tumblr selfies. Kogi single-origin coffee raw denim,
+              retro crucifix 90's tacos polaroid marfa wolf. Whatever disrupt normcore actually af. Roof party
```

Comment:
just to fix the annoying github syntax change here, can we kill this apostrophe?